components/omega/src/ocn/Eos.cpp

```diff
@@ -0,0 +1,253 @@
+#include "Eos.h"
+#include "DataTypes.h"
+#include "HorzMesh.h"
+
+namespace OMEGA {
+
+Eos *Eos::DefaultEos = nullptr;
+std::map<std::string, std::unique_ptr<Eos>> Eos::AllEos;
+
+TEOS10Poly75t::TEOS10Poly75t(int NVertLevels) : NVertLevels(NVertLevels) {
+   specVolPcoeffs = Array2DReal("specVolPcoeffs", 6, VecLength);
```

Comment:
Omega coding conventions should have the first letter capitalized here: SpecVolPCoeffs